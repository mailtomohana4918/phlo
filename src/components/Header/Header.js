import React from "react";
import { AppBar, Grid } from "@material-ui/core";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";

const Header = () => {
  return (
    <React.Fragment>
      <AppBar>
        <Toolbar variant="dense">
          <Grid container  justify="center">
            <Typography variant="h5" color="inherit">
              Appointment Booking
            </Typography>
          </Grid>
        </Toolbar>
      </AppBar>
    </React.Fragment>
  );
};
export default Header;
