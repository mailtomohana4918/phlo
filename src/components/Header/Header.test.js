import { render } from "@testing-library/react";
import React from "react";
import Header from "./Header";

describe("Header Component", () => {
  it("renders as expected", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
