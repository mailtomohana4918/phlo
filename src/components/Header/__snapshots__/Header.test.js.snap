// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders as expected 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
  >
    <div
      class="MuiToolbar-root MuiToolbar-dense MuiToolbar-gutters"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit"
        >
          Appointment Booking
        </h5>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
